src/hub.html

```diff
@@ -302,15 +313,27 @@
                                     position="-0.125 0.005 0.01"
                                 ></a-entity>
                             </a-entity>
-                            <a-entity mixin="rounded-button" is-remote-hover-target tags="singleActionButton: true;" class="hide-when-pinned" remove-networked-object-button position="0 -0.375 0.01">
+                            <a-entity
+                                mixin="rounded-button"
+                                is-remote-hover-target
+                                tags="singleActionButton: true;"
+                                hide-when-pinned-and-forbidden="whenPinned: true"
+                                remove-networked-object-button
+                                position="0 -0.375 0.01">
                                 <a-entity
                                     sprite="name: remote-action.png"
                                     icon-button="image: remote-action.png; hoverImage: remote-action.png"
```

Comment:
huh, `s/remote/remove/` here, not this PR but probably my typo if you feel like fixing :P